iocage_lib/ioc_start.py

```diff
@@ -1002,12 +1004,18 @@ def find_bridge_mtu(self, bridge):
             if dhcp == "on":
                 # Let's get the default vnet interface
                 default_if = self.get('vnet_default_interface')
-                if default_if == 'none':
+                if default_if == 'auto':
                     default_if = self.get_default_gateway()[1]
 
-                bridge_cmd = [
-                    "ifconfig", bridge, "create", "addm", default_if
-                ]
+                if default_if == 'none':
```

Comment:
Let's change this to `if default_if != 'none':` and drop the empty bridge creation, keeping only the else block under this.